### DISCUSSION TITLE
Update to Security Onion 2.3.60 kibana 404 page not found

### DISCUSSION THREAD

**Discussion by xpowershellx** [Security-Onion-Solutions/securityonion#4819] [Unsupported Versions]
I have a manager search and a forwarder .
All docker containers are running .

SOC is loading but no datas and when I try to open Kibana I get 404 page not found
Tried:
sudo so-kibana-config-load                            
Waiting for value 'Elastic' at 'http://localhost:5601/app/kibana' (1/300)
Server is not ready
Server still not ready after 300 attempts; giving up.
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (56) Recv failure: Connection reset by peer
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0 2395k    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (56) Recv failure: Connection reset by peer

Anyone can help ?

**Comment by pkgwillie** [comment]
I have same issue after update as well. Any help is much appreciated.

![kibana_config_reload](https://user-images.githubusercontent.com/7101154/125915024-c58eb36a-9fc3-4f40-96dc-2e07d79da0ce.JPG)

**Reply by pkgwillie** [reply]
@xpowershellx  is Kibana loading now? Kibana page is loading for me without any intervention after two days.

**Reply by DDD-Digithead** [reply]
Yep... same here... in fact after this most recent update the only things I can get to now are alerts and Grafana. Everything else renders a 404 error even though so-status reports everything is running.

Going reload... again. Seems to be the only way it's keep working after an update.

**Reply by TOoSmOotH** [reply]
Have any of you looked at the kibana log to see what the issue is?
